R/get_calib_init_validation.R

```diff
@@ -1,7 +1,7 @@
 #'@title Get last values from a calibration period to be used for the validation
 #'
 #'@description Reads in nml-file and output from calibration to get initial values for the calibration
-#'@param nml_file String of the GLM namelist file, in most cases this is 'glm3.nml'
+#'@param file_name String of the GLM namelist file, in most cases this is 'glm3.nml'
```

Comment:
Do you want to have this argument name differ from `read_nml`'s argument name?

Reply:
I'm unsure, I changed this function several times to finally make Travis pass all tests. There are now two args in the calibration: nml_file for the nml file that you want to work with (either glm3 or aed2), and glm_file that changes the time information in the glm setup. Maybe it makes more sense to change file_name back to glm_name to make everything more consistent.

Reply:
yeah, my suggestion is to make arguments that mean the same thing across functions be called the same thing, so `nml_file` in this case. Sometimes there isn't away around a conflict, so I understand if that may not be doable.

Reply:
Gotcha, I could change every instance of nml_file in the calibration to something like calib_file instead. My thinking is that these argument can either be the glm3.nml (if you want to calibrate a hydrodynamic parameter) or the aed2.nml (if you want to calibrate a wq parameter), so they are somehow different from the general nml_file. What you you think?

Reply:
I still think the `nml` file argument name should be consistent, since `read_nml` should (in the future) be able to read the aed2.nml just like it does for glm3 - both are nml files. But perhaps that is being idealistic - `calib_file` would be fine I think, but there is the potential for confusion as it could imply results file or an observation or parameters-to-modify file. But I may not have enough familiarity with this function to know if it is being used differently in here vs the other places where `nml_file` might show up in the package.